cli/flox/src/commands/pull.rs

```diff
@@ -290,7 +293,7 @@ impl Pull {
         force: bool,
         mut env: ManagedEnvironment,
         query_functions: Option<QueryFunctions>,
-    ) -> Result<()> {
+    ) -> Result<message::Action, anyhow::Error> {
```

Comment:
`anyhow::Result` defaults to `anyhow::Error` as the error variant.

```suggestion
    ) -> Result<message::Action> {
```